website/docs/r/security_partner_provider.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_partner_provider"
+description: |-
+  Manages a Security Partner Provider.
+---
+
+# azurerm_security_partner_provider
+
+Manages a Security Partner Provider.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_wan" "example" {
+  name                = "example-vwan"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_virtual_hub" "example" {
+  name                = "example-vhub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  virtual_wan_id      = azurerm_virtual_wan.example.id
+  address_prefix      = "10.0.2.0/24"
+}
+
+resource "azurerm_vpn_gateway" "example" {
+  name                = "example-vpngw"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  virtual_hub_id      = azurerm_virtual_hub.example.id
+}
+
+resource "azurerm_security_partner_provider" "example" {
+  name                   = "example-spp"
+  resource_group_name    = azurerm_resource_group.example.name
+  location               = azurerm_resource_group.example.location
+  virtual_hub_id         = azurerm_virtual_hub.example.id
+  security_provider_type = "IBoss"
+
+  tags = {
+    ENV = "Prod"
+  }
+
+  depends_on = [azurerm_vpn_gateway.example]
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Security Partner Provider. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Security Partner Provider should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the Security Partner Provider should exist. Changing this forces a new resource to be created.
+
+* `security_provider_type` - (Required) The security provider name. Possible values are `ZScaler`, `IBoss` and `Checkpoint` is allowed. Changing this forces a new resource to be created.
+
+* `virtual_hub_id` - (Optional) The ID of the Virtual Hub within which this Security Partner Provider should be created. Changing this forces a new resource to be created.
```

Comment:
What happens when no vhub is specified?

Reply:
The independent security partner provider would be created and it wouldn't be associated to virtual hub.